src/patterns/lambda-alarms.ts

```diff
@@ -0,0 +1,32 @@
+import { ComparisonOperator, MathExpression } from "@aws-cdk/aws-cloudwatch";
```

Comment:
I wasn't 100% sure whether this should be a `pattern` or a `construct` - any feedback on this would be helpful.

Reply:
My first impression is that this should be in constructs as patterns should essentially encompass an entire stack but interested to hear other people's opinions too

Reply:
Having read this comment and listened to @akash1810's excellent explanation during the presentations, I'm inclined to agree - I'll move this.

Reply:
Agree with @jamie-lynch and reminded that `InstanceRole` needs moving out of the patterns directory! https://docs.aws.amazon.com/cdk/latest/guide/constructs.html is a good read.